appdatabase/migrationsprevnodecfg/sql/1642666031_add_removed_clock_to_bookmarks.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE bookmarks ADD COLUMN removed BOOLEAN DEFAULT FALSE;
+ALTER TABLE bookmarks ADD COLUMN clock INT DEFAULT 0;
```

Comment:
This .sql migration should be placed in `appdatabase/migrations/sql/` instead of  `migrationsprevnodecfg`

Reply:
i tried this before and tested, looks like it will cause a new bug for status-react. if i place it into `appdatabase/migrations/sql/ ` instead of `migrationsprevnodecfg` , then u create a new account and login then logout then try login again, it will show u 
 an error `file not exist`, which is something with migration.

Reply:
here is the video when u place it in `appdatabase/migrations/sql/` instead of `migrationsprevnodecfg` @richard-ramos 
https://user-images.githubusercontent.com/12406719/151168523-a360370c-b59a-4185-8750-abb5eb7fd288.mp4

Reply:
it's another issue, but i would prefer to put it in `appdatabase/migrations/sql/` too. WDYT?

Reply:
Interesting! I'm going to try this PR on desktop and see what happens, will let you know in some minutes!

Reply:
Alright! found the issue with the migrations folders,  will do a fix and share the PR so you can rebase those changes! sorry!

Reply:
Here's the PR: https://github.com/status-im/status-go/pull/2504

Reply:
great, rebased!